### PR TITLE
Find command helpers.

### PR DESCRIPTION
**Pull request by sebv**
depends on https://github.com/appium/appium-base-driver/pull/15

ping @imurchie.

### CODE REVIEW

**Comment by sebv** [comment]
added getAttribute

File: lib/commands/element.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.getAttribute = async function (attribute, elementId) {
+  if (this.curContext) {
+    /* TODO */throw new errors.NotYetImplementedError();
+    // return await this.findWebElementOrElements(strategy, selector, mult, context);
+  } else {
+    if (_.contains(['label', 'name', 'value', 'values', 'hint'], attribute)) {
+      let command = ["au.getElement('", elementId, "').", attribute, "()"].join('');
```
Comment (imurchie) [inline lib/commands/element.js]: This sort of thing that we did in the old Appium would be much easier to read using string interpolation.

File: lib/commands/find.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  }
+  if (typeof context === "undefined" || !context) {
+    context = '';
+  } else if (typeof context === "string") {
+    context = util.escapeSpecialChars(context, "'");
+  }
+
+  // previously getSelectorForStrategy
+  if (strategy === 'class name' && selector.indexOf('UIA') !== 0) {
+    throw new errors.InvalidSelectorError("The class name selector must use full UIA class " +
```
Comment (imurchie) [inline lib/commands/find.js]: String interpolation.

File: lib/commands/index.js [hunk 3]
```diff
@@ -1,7 +1,11 @@
+import  findExtensions from './find';
+import  elementExtensions from './element';
```
Comment (imurchie) [inline lib/commands/index.js]: Spacing? Line 1 as well.

**Comment by sebv** [comment]
fixed stuff.

**Pull request merged**